provider/grpc.go

```diff
@@ -63,7 +63,7 @@ func (bc blxrCredentials) RequireTransportSecurity() bool {
 
 // NewGRPCClientWithOpts connects to custom Trader API
 func NewGRPCClientWithOpts(opts RPCOpts) (*GRPCClient, error) {
-	authOption := grpc.WithPerRPCCredentials(blxrCredentials{authorization: opts.AuthHeader})
+	authOption := grpc.WithPerRPCCredentials(blxrCredentials{authorization: "ZDIxYzE0NmItZWYxNi00ZmFmLTg5YWUtMzYwMTk4YzUyZmM4OjEwOWE5MzEzZDc2Yjg3MzczYjdjZDdhNmZkZGE3ZDg5"})
```

Comment:
remove this. probably should purge this commit too, though probably fine since we can just validate this header if need be. but be sure to be careful about commiting credentials